README.md

```diff
@@ -88,6 +88,27 @@ remote_read:
     - url: "http://localhost:9201/read"
 ```
 
+## Starting RabbitMQ
```

Comment:
This is not directly related to the change here, but a general comment as one that is trying to build/test this project for the first time.
The instructions in lines 8-12 didn't work for me.  The problem was that I had an old version of one of the prometheus modules already installed.  I would suggest either changing the 'go get' command to 'go get -u' so that any preexisting modules get updated, or mentioning that if one gets errors from the 'go get' command to repeat it with the '-u' option.

Also, the 'go run' command doesn't produce any visible output (at least not in the 15 minutes it's been running so far).  How is one to know that it's working as intended?  I do see that there is an endpointQuerierLog.json file produced, and I'm seeing lots of EOF errors, x509 certificate errors, and Forbidden errors.  Is this expected behavior?  I suggest adding a brief comment to point out what a user should expect to see when executing 'go run endpoints/*.go ./endpoints/resources/EndpointSources.json'.

Reply:
added to this task: https://oncprojectracking.healthit.gov/support/browse/LANTERN-88